values.js

```diff
@@ -1,25 +1,155 @@
 const values = {
-  'absolute': 'absolut',
+  // global values
+  'inherit': 'warisan',
+  'initial': 'awal',
+  'revert': 'balikkan',
```

Comment:
I think "kembalikan" is better

Reply:
If so, then 'warisan' should be 'wariskan' right? I'll change it.